envision/client.py

```diff
@@ -211,16 +201,18 @@ def run_socket(endpoint, num_retries, wait_between_retries, threadlocal):
                 )
 
                 if not connection_established:
-                    self._log.info(
-                        f"Attempting to connect to Envision tries={tries}/{num_retries}"
-                    )
+                    self._log.info(f"Attempting to connect to Envision tries={tries}")
+                else:
+                    # when connection closed, retry again every 5 seconds
+                    wait_between_retries = 5
+                    tries = 0
```

Comment:
I do not think resetting `tries` is needed. Or if you are resetting `tries` you will need to flip `connection_established` to false again.

```suggestion
                    self._log.info(f"Connection to Envision lost. Attempting to reconnect.")
```
```suggestion
                    tries = 0
                    self._log.info(f"Connection to Envision lost. Attempting to reconnect.")
                    connection_established = false
                    setattr(threadlocal, "connection_established", connection_established)
```